### PR TITLE
fix(lerna-semantic-release-analyze-commits): handle when affectsLine runs over multiple lines

### PR DESCRIPTION
**Pull request by bassjacob**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+6
```
Comment (lukebatchelor) [inline .nvmrc:1]: This is just to make it run in node 6 right?

**Comment by lukebatchelor** [comment]
LGTM!

File: .nvmrc (hunk 1 above)
Comment (bassjacob) [reply .nvmrc:1]: yep, exactly. mock-fs is only compatible with <=6, so if you cd into the dir and type `nvm use` it will grab a compatible node version

Comment (lukebatchelor) [reply .nvmrc:1]: Yep, misread the commit message, ignore me

**Comment by bassjacob** [comment]
@lukebatchelor @jpnelson I'll test this a little more. I've run it over all the commits in both projects we use internally and found a number of cases where this fix would release more packages than the old version.

I just want to try and see if I can catch any other edge cases.

**Pull request merged**